x-pack/plugins/secops/public/store/local/timeline/helpers.ts

```diff
@@ -488,7 +488,7 @@ export const updateTimelineDescription = ({
     ...timelineById,
     [id]: {
       ...timeline,
-      description,
+      description: description.endsWith(' ') ? `${description.trim()} ` : description.trim(),
```

Comment:
what's the goal here? is to trim and put back a space if there is one from the beginning

Reply:
It ensures the input never has more than one trailing space (including the case when you paste something from the clipboard that has multiple trailing spaces)